zerver/lib/actions.py

```diff
@@ -1004,6 +1006,7 @@ def get_recipient_info(recipient: Recipient,
         # of this function for different message types.
         assert(stream_topic is not None)
         user_ids_muting_topic = stream_topic.user_ids_muting_topic()
+        user_ids_muting_stream = stream_topic.get_user_ids_muting_stream()
```

Comment:
I don't think we need to do this database query; can't we just use is the `in_muted` data from `subscription_rows` just below?